app/src/main/java/com/android/shelfLife/ui/recipes/IndividualRecipe/IndividualRecipe.kt

```diff
@@ -197,9 +221,9 @@ private fun IndividualRecipeScreenPreviewEasterEgg() {
   IndividualRecipeScreen(navigationActions = navigationActions)
 }
 // this preview shows the example where we do have a selected recipe
-// @Preview()
-// @Composable
-// private fun IndividualRecipeScreenPreview() {
+//@Preview()
+//@Composable
```

Comment:
Is the preview really necessary? Why is it commented?

Reply:
its not, this relates to a previous issue. it shouldve been deleted a long time ago.